xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceHmacDigest.java

```diff
@@ -69,23 +60,6 @@ public String digestParams(RestInvocation restInvocation) {
       Mac mac = getMac();
       mac.update(input.getBytes("UTF-8"));
       String printBase64Binary = bytesToHex(mac.doFinal());
-
-      // https://github.com/mmazi/rescu/issues/62
-      // Seems rescu does not support ParamsDigest in QueryParam.
-      // hack to replace the signature in the invocation URL.
-      String invocationUrl = restInvocation.getInvocationUrl();
-      // String newInvocationUrl = UriBuilder.fromUri(invocationUrl).replaceQueryParam("signature",
-      // printBase64Binary).build().toString();
-
-      final String sig = "signature=";
```

Comment:
how are we passing this signature now without this code?

Reply:
Just like digesting other type of params, when the digested value (signature) is returned from this method it will become part of the query params. This is how the original digest interface is designed but historically it does not support Query Params as digest, as the rescu upgrade to support it, this can be simplified to just return the digested value